jenkins/revert-master.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -o errexit
+read -p "Delete master [press enter]"
+gcloud compute instances delete pull-jenkins-master
+read -p "Delete master disks [press enter]"
+gcloud compute disks delete pull-jenkins-master pull-jenkins-master-data pull-jenkins-master-docker
```

Comment:
what I'm worried about is if Jenkins falls over and someone tries to be helpful and runs this script, not fully understanding what it does, and inadvertently makes things worse.